site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -256,6 +256,55 @@ you can set the `.spec.resources[*].flavors[*].quota.max`
 If, for a given flavor, the `max` field is empty or null, a ClusterQueue can
 borrow up to the sum of min quotas from all the ClusterQueues in the cohort.
 
+## Preemption
+
+When there is not enough quota left in a ClusterQueue or its cohort, an incoming
+Workload can trigger preemption of previously admitted Workloads, based on
+policies for the ClusterQueue.
+
+A configuration for a ClusterQueue that enables preemption looks like the
+following:
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: team-a-cq
+spec:
+  preemption:
+    reclaimWithinCohort: Any
+    withinClusterQueue: LowerPriority
+```
+
+The fields above have the following semantics:
+
+- `reclaimWithinCohort` determines whether a pending Workload can preempt
+  Workloads from other ClusterQueues in the cohort that are using more than
+  their nominal quota. Possible values are:
+
+	  - `Never` (default): do not preempt workloads in the cohort.
```

Comment:
Not sure all this indentation space is needed

Reply:
removed